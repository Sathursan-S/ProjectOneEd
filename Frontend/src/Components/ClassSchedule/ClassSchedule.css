.ClassSchedule{
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;
    border: 2px solid #4b4a4d;
    width: 45vw;
    border-radius: 10px;
    height: 35px;
    margin-bottom: 1rem;

}
.schedule-end-time{
    display: flex;
    justify-content: flex-end;
    margin-left: auto;
}

.schedule-start-time{
    margin-left: auto;
    margin-right: auto;
}
.schedule-day{
    margin-right: auto;
}


@media screen and (max-width:1000px) {
    .ClassSchedule{
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;
    border: 2px solid #4b4a4d;
    width: 100%;
    border-radius: 10px;
    height: 35px;
    margin-bottom: 1rem;
}

.schedule-end-time{
    display: flex;
    justify-content: flex-end;
    margin-left: auto;
}

.schedule-start-time{
    margin-left: auto;
    margin-right: auto;
}
.schedule-day{
    margin-right: auto;
}

}