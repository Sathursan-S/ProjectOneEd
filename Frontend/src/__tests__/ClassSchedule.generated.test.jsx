import renderer from 'react-test-renderer';
import React from 'react'

import ClassSchedule from '../Components/ClassSchedule/ClassSchedule';

jest.mock('./ClassSchedule.css');

const renderTree = tree => renderer.create(tree);
describe('<ClassSchedule>', () => {
  it('should render component', () => {
    expect(renderTree(<ClassSchedule 
    />).toJSON()).toMatchSnapshot();
  });
  
});