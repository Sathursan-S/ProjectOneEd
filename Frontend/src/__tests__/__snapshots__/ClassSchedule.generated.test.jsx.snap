// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClassSchedule> should render component 1`] = `
<div
  className="ClassSchedule"
>
  <div
    className="schedule-day"
  >
    <span>
      Every
    </span>
    <span />
  </div>
  <div
    className="schedule-start-time"
  >
    <span>
      from
    </span>
    <span />
  </div>
  <div
    className="schedule-end-time"
  >
    <span>
      to
    </span>
    <span />
  </div>
</div>
`;
